lib/meadow/ingest/rows.ex

```diff
@@ -22,6 +22,16 @@ defmodule Meadow.Ingest.Rows do
     |> Repo.one()
   end
 
+  @doc """
+  Gets a row by ingest sheet and file set accession_number
+  """
+  def get_row_by_accession_number(sheet_id, file_set_accession_number) do
```

Comment:
Since there's already a `Rows.get_rows_by_work_accession_number/2`, can you name this `Rows.get_rows_by_file_set_accession_number/2` instead? I think a test for this is needed as well since we're expecting only one result, would be good to have that verified in a test.